pkg/solana/logpoller/query.go

```diff
@@ -31,6 +31,12 @@ func (q *queryArgs) withField(fieldName string, value any) *queryArgs {
 	return args
 }
 
```

Comment:
I just noticed there is a `parser_test.go` but no `query_test.go`, while in evm logpoller we have both. Are the functions in `query.go` unit tested anywhere?

Seems like we should have unit tests for them, but not sure whether they should go in this PR or in @dhaidashenko 's db models PR it's based on.

Reply:
Update: I think this probably belongs in the other PR, but let me know what you both think.

Reply:
They are tested, but indirectly, the DB schema PR provides 70% coverage. Combined with this PR, it should be even higher. Given the lack of complex logic, I'm not sure dedicated tests are needed at this stage.